WIREDictionary_test.go

```diff
@@ -1,7 +1,3 @@
-// Copyright 2019 The Moov Authors
-// Use of this source code is governed by an Apache License
-// license that can be found in the LICENSE file.
-
```

Comment:
Can we add the license header back in?

Reply:
Yep, I'll add it with the next commit to the repo